test/functional/apps/discover/group3/_lens_vis.ts

```diff
@@ -110,8 +110,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     return seriesType;
   }
 
-  // FLAKY: https://github.com/elastic/kibana/issues/184600
-  describe.skip('discover lens vis', function () {
+  describe('discover lens vis', function () {
```

Comment:
there's some really dark magic here, that even when investigating this caused my IDE to freeze, so just before the last meeting of my day, I'm thinking that the cleanup of adhoc data views that were changed because of a spec change, should be triggered later on:

https://github.com/elastic/kibana/blob/7f1503e108148e30180f718f86c45e1f32266684/src/plugins/discover/public/application/main/state_management/discover_state.ts#L430

because then the previous data view would still be available, wouldn't do any harm, and be removed after the data fetching was triggered

Reply:
clearly broken here, looking for a simpler solution, so far I couldn't this seems to resolve this issue locally, let's see what flaky test runner says: #199982